purchase_packaging_default/models/purchase_order_line.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2024 Camptocamp (<https://www.camptocamp.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import math
+
+from odoo import api, fields, models
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = "purchase.order.line"
+
+    def _get_default_packaging(self):
+        """From product get 1st packaging found ordered by sequence"""
+        product_template = self.product_id.product_tmpl_id
+        return fields.first(product_template.packaging_ids)
+
+    @api.onchange("product_id")
+    def _onchange_product_id(self):
+        if self.env.user.company_id.purchase_packaging_default_enabled:
```

Comment:
```suggestion
        if self.env.company.purchase_packaging_default_enabled:
```
non blocking of course